frontend/src/main.js

```diff
@@ -3,4 +3,5 @@ import App from './App.vue'
 import { router } from './router'
 
 const app = createApp(App).use(router);
+app.config.unwrapInjectedRef = true;
```

Comment:
What's this for?🤔

Reply:
When I added the `provide`/`inject` elements I struggled a bit, because it did not work as expected. It turned out that the injected object is wrapped and needs explicit unwrapping in the current version of Vue. But it is planned to automatically unwrap it in a future release. 

This option enables this behavior in the current version.